### PR TITLE
chore: upgrade go version to 1.19

### PR DESCRIPTION
**Pull request by sudiptob2**
related to [related to jenkins-x/jx#8433](https://github.com/jenkins-x/jx/issues/8433)

### CODE REVIEW

**Review by tomhobson — APPROVED** [review]

File: .github/workflows/jenkins-x-pr.yaml [hunk 1]
```diff
@@ -14,14 +14,14 @@ jobs:
     - env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       name: build-make-linux
-      uses: docker://golang:1.18
+      uses: docker://golang:1.19
```
Comment (ankitm123) [inline .github/workflows/jenkins-x-pr.yaml:17]: Be more specific about the tags, use 1.19.5 (We did not do it in the past, but it's never late to do the right thing :grimacing: )

Comment (sudiptob2) [reply .github/workflows/jenkins-x-pr.yaml:17]: Thanks for pointing it out, Honestly, I also thought about it but I took examples from other plugins that have been upgraded. Will accommodate the change.

Comment (sudiptob2) [reply .github/workflows/jenkins-x-pr.yaml:17]: I tried to pin the go version to `1.19.5` However I am getting error while doing so.

```bash
go: errors parsing go.mod:
jx-pipeline/go.mod:195: invalid go version '1.19.5': must match format 1.23
```

Comment (ankitm123) [reply .github/workflows/jenkins-x-pr.yaml:17]: The pinning is only for the github actions file. For go.mod, you can only do `1.19`

Comment (sudiptob2) [reply .github/workflows/jenkins-x-pr.yaml:17]: Thanks much, understood

**Review by ankitm123 — APPROVED** [review]

**Pull request merged**